service-catalog/components/opf-monitoring-grafana-operator.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: opf-monitoring-grafana-operator
+  title: Grafana operator
+  description: A Kubernetes Operator based on the Operator SDK for creating and managing Grafana instances
+  annotations:
+    operate-first.cloud/logo-url: https://raw.githubusercontent.com/grafana/grafana/main/public/img/grafana_icon.svg
+    argocd/app-name: grafana-smaug
+    github.com/project-slug: operate-first/apps
+    backstage.io/kubernetes-label-selector: "control-plane=controller-manager"
+    backstage.io/kubernetes-namespace: opf-monitoring
+    backstage.io/techdocs-ref: url:https://github.com/operate-first/apps/tree/master/docs/content/grafana
+  links:
+    - url: https://github.com/operate-first/apps/issues/new
```

Comment:
to avoid duplication and maybe links going stale, should we add most links at the `systems` level?
probably just a detail and nitpick